sphinx/writers/latex.py

```diff
@@ -368,6 +369,17 @@ def __init__(self, document, builder):
             'releasename':  _('Release'),
             'indexname':    _('Index'),
         })
+        if builder.config.latex_engine == 'lualatex':
+            self.elements['luatex'] = \
```

Comment:
good idea. User customizing template will thus avoid unneeded checks about luatex85 once her TeX install is up to date.